src/Dibi/Drivers/OracleResult.php

```diff
@@ -111,7 +111,7 @@ public function getResultColumns(): array
 	public function getResultResource()
 	{
 		$this->autoFree = false;
-		return is_resource($this->resultSet) ? $this->resultSet : null;
+		return $this->resultSet;
```

Comment:
it changes behavior

Reply:
Value of property `$resultSet` is always `resource`, because only `__construct` can write value and default value always exist.

![Snímek obrazovky 2020-05-07 v 22 41 42](https://user-images.githubusercontent.com/4738758/81342728-fc05b680-90b3-11ea-820b-956759608d9c.png)

Reply:
Reverted.